spec/models/coronavirus_page_spec.rb

```diff
@@ -15,4 +15,14 @@
       expect(CoronavirusPage.subtopic_pages).to eq [business, education, employees]
     end
   end
+
+  describe "dependencies" do
+    let!(:employees) { create :coronavirus_page, :employees }
+    let!(:sub_section) { create :sub_section, coronavirus_page: employees }
+
+    it "deletion destroys all child subsections" do
+      employees = CoronavirusPage.find_by(slug: "employees")
```

Comment:
Is this line needed as `employees` is already available via the `let!(:employees)` declaration on line 29.

Reply:
Thanks 👍